istio.VERSION

```diff
@@ -4,11 +4,12 @@ export CA_HUB="gcr.io/istio-testing"
 export CA_TAG="b7b8c10bbb6ca471e9c8aee2b3889f0ccbf01c3c"
 export MIXER_HUB="gcr.io/istio-testing"
 export MIXER_TAG="4e107a2411e1d58e88fd9ae4d32a139751e87955"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/istioctl"
 export PILOT_HUB="gcr.io/istio-testing"
 export PILOT_TAG="3ac357f84576f881d9f57ed76e4e14a052ac5333"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/istioctl"
 export ISTIO_NAMESPACE="istio-system"
 export AUTH_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/auth/b7b8c10bbb6ca471e9c8aee2b3889f0ccbf01c3c/artifacts/debs"
 export PILOT_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/pilot/3ac357f84576f881d9f57ed76e4e14a052ac5333/artifacts/debs"
-export PROXY_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/proxy/4a493c07e34e764e29013c794ad295f665d8eebc/artifacts/debs"
-
+export PROXY_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/proxy//artifacts/debs"
+export FORTIO_HUB="gcr.io/istio-testing"
+export FORTIO_TAG="a9b5395d6fe2e8cbcbe9b0835d0b48e88620b878"
```

Comment:
this tag is from a bot presubmit, the artefact building is missing so far from postsubmit (separate pr)